packages/graphback-datasync/src/providers/DatasyncMongoDBDataProvider.ts

```diff
@@ -0,0 +1,53 @@
+import { GraphQLObjectType } from 'graphql';
+import { NoDataError, FieldTransform, TransformType } from '@graphback/runtime';
+import { getDatabaseArguments } from '@graphback/core';
+import { ObjectId } from 'mongodb';
+import { MongoDBDataProvider } from '@graphback/runtime-mongo';
+
+/**
+ * Mongo provider that contains special handlers for offix conflict resolution format:
+ *
+ * https://offix.dev/docs/conflict-server#structure-of-the-conflict-error
+ */
+export class DatasyncMongoDBDataProvider<Type = any, GraphbackContext = any> extends MongoDBDataProvider<Type, GraphbackContext> {
```

Comment:
DataSync

Reply:
I am a bit puzzled as to what to do here, do you mean to change docstring or name of class

Reply:
I ment rename. You can rename to camelcase. We can do it afterwards.